### DISCUSSION TITLE
Support for tagging pages& search based on tag filters

### DISCUSSION THREAD

**Discussion by Sharonkenny222** [GitbookIO/community#112] [Feature Requests]
### What's your feature/idea?

we have identified 2 key features that are not available that we are very keen to see on the platform : 
Tagging of pages / content so we can have a way to identify type of content 

Allowing an 'advanced search' functionality that would either allow our users to filter by type of content (based on filtering by tag) and / or by extending to multiple spaces? 

### What's your desired outcome?

Help us identify the type of content and easier search for users.

### What's the impact of this for you?

- [ ] Critical
- [X] High
- [ ] Medium
- [ ] Low

**Comment by kristenoneill** [comment]
Would love to see this as well - We'd ideally like to tag content blocks within a page so that users could search and filter. We are using a page to stream product updates. Although the side navigation panel is helpful - it would be great if our users could filter the content based on the tag to see the relevant updates for them. Not sure if this would be possible without adding blocks/sections.

**Comment by maguerin** [comment]
Page level tagging + advanced search which would allow to surface and filer by tag would be a generic and nice feature.

**Comment by kb-1** [comment]
We are "hacking" this right now with cards and multi-select on every page (but its a really bad hack). For content that could be categorized in multiple dimensions, tagging is a must!

**Comment by denisemcg** [comment]
I need this, too. I have multiple products/modules on one platform. Some of these products have similar content/topics, e.g. export data. When I search for a keyword, I might only be interested in one module, but I don’t know from the list of search results if a page is relevant to me or not. Currently, my workaround it to manually add the module name into the page title, which is not ideal.

**Comment by drewkerry19** [comment]
We really could use this feature at well.  We would love to be able to tag pages or sections within pages with various topic/category tags, and then allow uses to filter content based on those tags.  For example, our salespeople could filter the changelog to only project updates related to sales projects.  Our operations people could filter based on projects related to certain features or departments, etc.  This would be immensely helpful and is the major navigation/organization issue that is plaguing our spaces right now.

**Comment by kcookrose** [comment]
My company is very interested in this as well. Our use case: We'd like to add keywords to a page to help people searching for the wrong words. For example, we have documentation about our Materials. But people coming from other 3D software platforms probably won't be searching for that term; they'll be searching for terms like "swatches" or "tiles." I'd like to be able to add keywords to our Materials page so if someone searching for swatches, the Materials page would pop up.
